app/controllers/admin/creator_settings_controller.rb

```diff
@@ -4,34 +4,33 @@ class CreatorSettingsController < Admin::ApplicationController
                         invite_only_mode logo primary_brand_color_hex public].freeze
 
     def new
+      @creator_settings_form = CreatorSettingsForm.new(
+        community_name: ::Settings::Community.community_name,
+        public: ::Settings::UserExperience.public,
+        invite_only_mode: ::Settings::Authentication.invite_only_mode,
+        primary_brand_color_hex: ::Settings::UserExperience.primary_brand_color_hex,
+      )
       @max_file_size = LogoUploader::MAX_FILE_SIZE
       @logo_allowed_types = (LogoUploader::CONTENT_TYPE_ALLOWLIST +
         LogoUploader::EXTENSION_ALLOWLIST.map { |extension| ".#{extension}" }).join(",")
     end
 
     def create
       extra_authorization
-      ActiveRecord::Base.transaction do
```

Comment:
You'll notice that we completely remove `ActiveRecord::Base.transaction do`. I chatted to @citizen428 about this and we concluded that each of those Settings are not directly dependant on each other, so it may be okay to save some and to then allow them to update others afterwards if it fails.